src/Core/Services/CryptoService.cs

```diff
@@ -9,6 +9,7 @@
 using Bit.Core.Models.Domain;
 using Bit.Core.Models.Response;
 using Bit.Core.Utilities;
+using static System.Net.WebRequestMethods;
```

Comment:
remove?

Reply:
Yeah... seems Visual studio snuck in some unused imports.

Reply:
Fixed in e765f2de659d219f34b19c6523bf92baa958f809